spec.bs

```diff
@@ -2731,13 +2735,21 @@ a {{ReportingBrowserSignals}} |browserSignals|, a [=direct from seller signals=]
      [=generated bid/interest group=]'s [=interest group/ads=] whose
      [=interest group ad/render url=] is |winner|'s
      [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
-  1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-     does not [=map/exist=] and the result of [=query reporting ID k-anonymity count=] given
-     |winner|'s [=generated bid/interest group=] and |igAd| is true:
-    1. If |igAd|'s [=interest group ad/buyer reporting ID=] [=map/exists=],
+  1. If |winner|'s [=generated bid/selected buyer and seller reporting ID=] is not null:

```

Comment:
Similarly, could we add notes here ...

```
  1. If |winner|'s [=generated bid/selected buyer and seller reporting ID=] is not null:
  
      Note: If the winning ad has a selectedBuyerAndSellerReportingId, it would have already been
      set by the preceding call to reportResult, and provided to this algorithm in the
      ReportingBrowserSignals returned by that algorithm and provided to this one.
```